src/api/common/index.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   dropsToXrp: utils.dropsToXrp,
   xrpToDrops: utils.xrpToDrops,
   toRippledAmount: utils.toRippledAmount,
-  generateWallet: utils.generateWallet,
+  generateAddress: utils.generateAddress,
```

Comment:
I mean in the general sense. Here where it's generating an accountID and a
secret it feels somehow better to use generateAddress. Only alternative
name that pops to mind is generateAccountKeys which I guess would make
sense when RegularKey joins the party. But then, "just use the accountID in
the RegularKey field" ...